### ISSUE TITLE
Minimap is broken

### ISSUE THREAD

**Issue by TheMegaGamer** [Ishigh1/AdvancedSeedGen#9]
I have a world of size 15,000 x 5,000, and the minimap breaks at the edges.

**Comment by Yorunetsu** [comment]
Yeah, can confirm, map breaks both vertically and horizontally at about ~500 feet to the edge, but only to the right and down, up and left are okay. My world size is 16800x4800, so, completely within the vanilla proportions.

![image](https://user-images.githubusercontent.com/89686166/176188055-796646a7-0c1b-4751-850d-d2b98276fee4.png)![image](https://user-images.githubusercontent.com/89686166/176188613-ffb3b30b-7ec4-4976-96da-6fc944f14777.png)
